src/index.ts

```diff
@@ -145,6 +145,15 @@ export class PornHub {
         return modelPage(this.engine, urlOrName)
     }
 
+    /**
+     * Get list of model's uploaded videos
+     * @param urlOrName Model name or page url
+     * @param pageNumber Which page of video list to request (defaults to 1)
+     */
+    modelVideos(urlOrName: string, pageNumber = 1) {
```

Comment:
Let's move the `pageNumber` into a config object, and the name should also be renamed to `page`

Reply:
Created #76